addons/test_mail/tests/test_performance.py

```diff
@@ -836,23 +836,14 @@ def test_message_format(self):
     @users('emp')
     @warmup
     def test_message_format_query(self):
-        name_field = self.env['ir.model.fields']._get(self.container._name, 'name')
-        customer_id_field = self.env['ir.model.fields']._get(self.container._name, 'customer_id')
-
         record1 = self.env['mail.test.simple'].create({'name': 'Test1'})
         record2 = self.env['mail.test.simple'].create({'name': 'Test2'})
         record3 = self.env['mail.test.track'].create({'name': 'Test3'})
 
-        messages = self.env['mail.message'].sudo().create([{
-            'model': 'mail.test.simple',
-            'res_id': record1.id,
-        }, {
-            'model': 'mail.test.simple',
-            'res_id': record2.id,
-        }, {
-            'model': 'mail.test.track',
-            'res_id': record3.id,
-        }])
+        messages = self.env['mail.message'].create([{
+            'model': record._name,
+            'res_id': record.id
+        } for record in [record1, record2, record3]])
```

Comment:
you would want to apply all of these changes in your first commit introducing the test, not in the second commit that should just adapt the query count if need be

Reply:
My bad, I amended without thinking twice...